### ISSUE TITLE
Browser overlay does not display data

### ISSUE THREAD

**Issue by wompapmow** [EvanPurkhiser/prolink-tools#32]
Hello,

First I just want to say thank you, I've been watching this for years and this is an amazing tool. I am finally moving ahead with trying it out and integrating it into my livestreams.

Using the latest build (0.1.0-prerelease.11), I cannot get any data to display in the browser. My devices reflect properly in the main 'Device status' window, show accurate track data and phase status, etc. I am using CDJ-3000s.

When I go to configure/create an overlay, I cannot get it to display 'live' data - it only displays the watermark for DEMO data. Even though the device status reflects players connected and accurate data. I have tried restarting the song during scene configuration and it never hooked.

I can access the locally hosted server through my browser and it's fine - just devoid of data, empty screen and only a single JS file in the source.

I am happy to provide additional data/logs/screenshots if necessary, I feel like there's a simple fix (assuming the implementation of overlay functionality is finished/working).

Cheers!

**Comment by wompapmow** [comment]
I see from issue #24 that "there is a bug currently in the OBS overlay preventing track information there" - is this still the case? That post was from August, so I'm not sure if this is the same bug referenced.

**Comment by EvanPurkhiser** [comment]
What mixer are you using?

I built the now-playing logic with a pretty heavy dependency on the 'on-air' state of the CDJ (the red ring when the fader is up). The 900nxs and up mixers all report on-air state to the CDJs, but you have to make sure the mixer is on the network.

**Comment by wompapmow** [comment]
I'm using a V10. I wasn't testing previously using the mixer but I tried again with it online and no real change. I've also tried testing with my old XDJ1000-MK2, same behavior. I believe it sees the player as On-Air (UI element turns red) and it shows the correct track data in the Device Status window.

I can try a few more permutations, unfortunately the V10 is the only mixer I have to test with. I see that you recently added data collection for debugging, any way I can access that information? https://github.com/EvanPurkhiser/prolink-tools/commit/b85ff2c3f9abeaace320cec4f31b0a502ee80525
